app/scripts/views/table.js

```diff
@@ -14,8 +14,8 @@ default class TableView extends Component {
     return (
       <Table responsive>
         <tbody>
-          {this.props.table.map(val => {
-            return <tr><td>{addr(val)}</td></tr>
+          {this.props.table.map((val, i) => {
+            return <tr key={i}><td key={i}>{addr(val)}</td></tr>
```

Comment:
I added key attributes here because react needs them in order to find the appropriate DOM nodes inside an array without re-rendering the entire view.